lib/config/definitions.js

```diff
@@ -1397,7 +1397,13 @@ const options = [
     stage: 'package',
     type: 'json',
     default: {
-      fileMatch: ['\\.csproj$'],
+      fileMatch: ['(^|/)*\\.csproj$'],
```

Comment:
What are you trying to achieve with this change?

Reply:
Nothing, this is not a part of request. I will revert it.

Reply:
Reverted